docs/js/contribution.md

```diff
@@ -0,0 +1,148 @@
+
+### Introducing dependencies
+
+    - Dependency audit should happen as a part of PR review
+    - SDK provides a guide on dependency choices to the contributing team
+
+### Coding convention
+
+    - Contribution team should use coding convention of the SDK unless it's explicitly approved to be otherwise
+    - Most of the coding convention rules should be enforced by automated code checkers and linters like ESLint and Prettier
+    - Enforcement happens via continuous integration pipeline
+    - SDK team provides guidelines on coding conventions that can't be automatically enforced
+    - SDK team shares documentation on standard patterns chosen by SDK for ... (what patterns do we use?)
+
+### Pull requests
+
+    - Pull request should contain some logical and complete pile of work
+    - Pull request should be **small**  for easy review
+    - Preferably Pull request should contain a tangible incremental value
+    - Pull request are mandatorily reviewed by at least one of SDK team members
+
+### Branching strategy
+
+  - `Main` branch has to be always **release ready**. In other words: up to date, tested, and documented
+
+## Documentation
+
+SAP Cloud SDK for JavaScript has two types of documentation:
+
+    - Documentation portal
+    - Generated API documentation
+
+### Documentation portal
+
+    - After reaching `Beta`, feature or API library has to be documented by contribution team on SDK's documentation portal
+    - Documentation has to be kept up to date by the contribution team
+
+### Generated API documentation
+
+    - API documentation has to be generated
+    - SDK team provides means to generate API documentation on the available codebase
+
+### README
+
+    - Contribution team maintains `README` on the contribution repository according to development needs
+
+## Releases
+
+### Release process
+
+    - Every library released with SDK should use [semantic versioning](https://semver.org/
+    - Breaking API changes have to be avoided.
+    - If breaking change is introduced, the old API must be marked deprecated instead of removal. A grace period of 6 months to be observed before removal can be considered.
+    - The contribution team has to set on a regular release cycle. SDK uses 2 weeks released cycle and it's preferred regularity.
+    - All the new features have to be released as `Beta` first. `General availability` release is triggered when API is stabilized and no breaking changes are planned.
+    - If a new module is to be released, its naming has to be aligned with the SDK team
+    - Practicalities of releasing NPM module into SDK's namespace are provided by SDK team
+    - Release change-log has to be maintained by the contribution team
+
+
+### License
+
+    - Open-source modules of Cloud SDK for Java are licensed under Apache 2.0
+    - The contribution must have the same license applied
+    - Other cases have to be explicitly discussed and approved by the SDK team
```

Comment:
We cannot approve this. All SAP Open source products libs are under Apache-2.0 as far as I know. I would not mention it.